ivy_tests/test_ivy/test_functional/test_experimental/test_nn/test_norms.py

```diff
@@ -141,3 +141,73 @@ def test_instance_norm(
         affine=affine,
         track_running_stats=track_running_stats,
     )
+
+
+@handle_test(
+    fn_tree="functional.ivy.experimental.l2_normalize",
+    dtype_and_x=helpers.arrays_and_axes(
+        available_dtypes=helpers.get_dtypes("float"),
+        num=1,
+        returndtype=True,
+        force_int_axis=True,
+    ),
+    test_gradients=st.just(False),
+)
+def test_l2_normalize(
+    *,
+    dtype_and_x,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    input_dtype, x, axis = dtype_and_x
+    helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        on_device=on_device,
+        fw=backend_fw,
+        fn_name=fn_name,
+        rtol_=1e-1,
+        atol_=1e-1,
+        x=x[0],
+        axis=axis,
+    )
+
+#lp_normalize
+@handle_test(
+    fn_tree="functional.ivy.experimental.lp_normalize",
+    dtype_and_x=helpers.arrays_and_axes(
+        available_dtypes=helpers.get_dtypes("float"),
+        num=1,
+        returndtype=True,
+        force_int_axis=True,
+    ),
+    test_gradients=st.just(False),
+)
+def test_lp_normalize(
+    *,
+    dtype_and_x,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+    
+):
+    input_dtype, x, axis = dtype_and_x
+    helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        on_device=on_device,
+        fw=backend_fw,
+        fn_name=fn_name,
+        rtol_=1e-1,
+        atol_=1e-1,
+        x=x[0],
+        axis=axis,
+        p=2,
+    )
```

Comment:
Move this test to test_experimental/test_core/test_norms. Also generate a float for p instead of just passing 2.